lambda-packages/role_handler/index.js

```diff
@@ -156,11 +156,84 @@ const generateRoleName = exports.generateRoleName = function (logicalResourceId)
   return roleName;
 };
 
-const deleteRole = async function (roleName) {
+const updateRole = async function (roleName, props, oldProps) {
+  const iam = new aws.IAM();
+
+  if (waiter) {
+    // Used by the test suite, since waiters aren't mockable yet
+    iam.waitFor = waiter;
+  }
+
+  console.log(`Updating role ${roleName}...`);
+
+  const {
+    Role
+  } = await iam.getRole({
+    RoleName: props.RoleName
+  }).promise();
+
+  let updateRoleProps = {}
+  if (props.Description) {
```

Comment:
don't use bare values in `if` predicates since they are ambiguous - use `if (props.Description !== undefined)` etc